static/js/webrtc.js

```diff
@@ -168,13 +168,17 @@ var rtc = (function() {
         var videoTrack = localStream.getVideoTracks()[0];
         var audioTrack = localStream.getAudioTracks()[0];
         self.setStream(self._pad.getUserId(), "");
-        if (videoTrack.stop === undefined) {
+        if ((videoTrack && videoTrack.stop === undefined) || (audioTrack && audioTrack.stop === undefined)) {
```

Comment:
`videoTrack` and `audioTrack` may now legitimately not exist if they are `"hard"` disabled.